chain-core/src/init/network.rs

```diff
@@ -40,6 +42,14 @@ fn init_network(network: Network) {
     }
 }
 
+#[cfg(test)]
```

Comment:
why is this test-only `init_network` necessary?

Reply:
because for testing, need to change NETWORK in runtime,
that's not allowed.
so used conditional compilation.